homeassistant/components/voip/voip.py

```diff
@@ -233,64 +224,34 @@ async def _run_pipeline(
             await self._play_listening_tone()
 
         try:
-            # Wait for speech before starting pipeline
-            segmenter = VoiceCommandSegmenter(silence_seconds=self.silence_seconds)
-            audio_enhancer = MicroVadEnhancer(0, 0, True)
-            chunk_buffer: deque[bytes] = deque(
-                maxlen=self.buffered_chunks_before_speech,
-            )
-            speech_detected = await self._wait_for_speech(
-                segmenter,
-                audio_enhancer,
-                chunk_buffer,
+            await self.satellite.async_set_config(
+                assist_satellite.SatelliteConfig(
+                    default_pipeline=pipeline_select.get_chosen_pipeline(
```

Comment:
For a future PR, the assist satellite base class needs to handle this

Reply:
Config has been removed for now